inst/schema/spec.md

```diff
@@ -1,8 +1,173 @@
 
-# Data
-## POST /data/impact
-Accepts a set of baseline options and generates a data set 
+# Graphs
+
+## POST /graph/prevalence/data
+Accepts a set of baseline options and generates a data set for the prevalence graph
+
+Request schema: [DataOptions.schema.json](./DataOptions.schema.json)
+
+Return schema: [Data.schema.json](./Data.schema.json)
+
+### Example
+#### Request
+```
+{
+  "irs_future": "80%",
+  "sprayInput": 1,
+  "biting_indoors": "high",
+  "population": 1000
+}
+```
+
+#### Response
+```
+[
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "low",
+    "resistance": 0.2,
+    "value": 0.4287
+  },
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "low",
+    "resistance": 0.4,
+    "value": 0.4119
+  },
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "low",
+    "resistance": 0.6,
+    "value": 0.6894
+  },
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "low",
+    "resistance": 0.8,
+    "value": 0.6217906241600673
+  },
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "med",
+    "resistance": 0.2,
+    "value": 0.4716
+  },
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "med",
+    "resistance": 0.4,
+    "value": 0.8187
+  },
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "med",
+    "resistance": 0.6,
+    "value": 0.0453
+  },
+  {
+    "month": 0,
+    "intervention": "none",
+    "net_use": 0,
+    "irs_use": 0,
+    "prevalence": "med",
+    "resistance": 0.8,
+    "value": 0.1402
+  },
+  ...
+]
+```
+
+## GET /graph/prevalence/config
+Returns an array of graph configuration object for the prevalence graph.
+
+Properties:
+* an id
+* a plotly layout object
+* a data object containing all the plotly metadata for the data series as well as the columns defining the series
+
+and optionally:
+* a plotly config object
+
+Schema: [Graphs.schema.json](./Graphs.schema.json)
+
+Plotly documentation: https://plotly.com/javascript/plotly-fundamentals/
+
+### Example 
+```
+{
+    "id": "prev",
+    "layout": {
+           "title": "Projected prevalence in under 5 year olds",
+           "xaxis": {
+               "title": 'years of intervention',
+               "showline": true,
+               "tickvals": ["prev_year_1", "prev_year_2", "prev_year_3"],
+               "ticktext": [1, 2, 3],
+           },
+           "yaxis": {
+               "title": 'prevalence (%)',
+               "showline": true,
+               "range": [0, 100],
+               "autorange": false
+           }
+    },
+    "data": [
+         {
+             cols: ["prev_year_1", "prev_year_2", "prev_year_3"],
```

Comment:
These cols no longer make sense with the long format months data, will fiddle around with the prototype to change this